src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -88,6 +88,12 @@ class ClusterObserver : IDisposable
                     if (Suspended == false)
                     {
                         _iteration++;
+                        if (_iteration == 1)
+                        {
+                            prevStats = _maintenance.GetStats();
```

Comment:
Why aren't you doing that before the `while`?

Reply:
done. 
Thought to have it inside the try/catch but `GetStats()` can't really throw so moved it before the `while`.